### PR TITLE
Update doc on  Ansible Collection and manual installation steps

### PR DESCRIPTION
**Pull request by rpelisse**
@cmoulliard Here the manual steps to install JCliff, feel free to review and coments!

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -18,13 +18,19 @@ JCliff modifies the configuration incrementally. Only necessary parts
...
-### How do I use puppet?
+### How do I use Puppet?
 
 Use puppet to lay down configuration files, and then have puppet
 execute jcliff. This way, you can define datasources, logging,
 etc. using puppet code and templates, and configure only what's
 necessary.
 
+### How od I use it with Ansible
+
```
Comment (bserdar) [inline README.markdown:29]: Fix typo pls: od -> do

Comment (rpelisse) [reply README.markdown:29]: @bserdar fixed!

**Comment by cmoulliard** [comment]
Project cannot be compiled on macos as maven during the execution of `mvn install` command reports the following `sed` error:

```
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ jcliff ---
[INFO] 
[INFO] --- exec-maven-plugin:1.4.0:exec (default) @ jcliff ---
sed: 1: "s/(String VERSION=)".*" ...": \1 not defined in the RE
[INFO] ------------------------------------------------------------------------
```
@rpelisse @bserdar

**Comment by rpelisse** [comment]
@cmoulliard I think you may be able to get away with simply running $ maven package (instead of install) if you want this to work on MacOSX. Otherwise, simply run this build inside a Docker container based on any Linux.

**Comment by cmoulliard** [comment]
> $ maven package (instead of install)

That fails too and generates `sed: 1: "s/(String VERSION=)".*" ...": \1 not defined in the RE`

**Comment by cmoulliard** [comment]
Trick is to build the jar using `mvn jar:jar` ;-)

**Comment by cmoulliard** [comment]
We cannot run `jcliff` bash script as it is on a macos machine

```
 ./src/main/scripts/jcliff                                    
./src/main/scripts/jcliff: line 69: @: unbound variable
```

**Comment by sabre1041** [comment]
@cmoulliard you need to pass arguments to the `jcliff` command. if you run `jcliff -v` it will show a successful result

**Comment by rpelisse** [comment]
@cmoulliard that has NOTHING to do with MacOS ;) ! Look at the error message, it's actually Bash complaining that there is an unbound variables ! 

Which probably means that something the script expect to be defined (like it JCLIFF_HOME which is defined if you install the RPM) is not defined.  Given that the content of line 69, it seems the variable JAVA has not been defined, which probably means that JAVA_HOME is not being properly defined.

**Comment by rpelisse** [comment]
I'll add a note in the README to say that one need to look into the script and set env vars. I don't want to enumerate the env vars that need to be set because, then you need to update the documentation every time the scripts changes. I would rather keep the script as the "unique source of thruth".

**Pull request merged**